_pages/interviews.md

```diff
@@ -39,11 +40,18 @@ Now, **this isn’t an excuse to be a jerk!** Being selective is important, but
 
 - **If you come to a conclusion very quickly, in the first few minutes of the interview, spend the rest of the interview trying to disprove that snap judgement.** Snap judgement are much more likely to be prone to bias than considered ones. We tend to jump to conclusions and then look for evidence to support our hypothesis. To compensate, if you find you’ve reached a Yes/No conclusion very quickly, spend the rest of the interview trying to disprove that hypothesis. Explicitly look for evidence that you’re wrong. If you’ve decided immediately that a candidate is not qualified, ask targeted questions and spend rest of the interview trying as hard as you can to find evidence that they are qualified.
 
+<a name="before-you-start"></a>
+## Before you start the interview
+
+- Before you begin each interview, [remind yourself to watch out for unconscious bias]({{site.basurl}}/unconcious-bias). Remember that we’re especially susceptible to assume that underrepresented minorities -- women, people of color, etc. -- are less qualified than their white male counterparts. Be wary of “gut decisions” that aren’t backed up by specific behavior you’ve noted during the interview (this is one reason we ask behavioral questions, and try to take notes behaviorally).
+
+- Spend some time becoming familiar with the candidates resume, which is always attached to calendar events.
+
+- Review any other links provided with the resume, e.g. Github and LinkedIn.
+
```

Comment:
- Do not look up the candidate on social media sites that have not provided.

Reply:
Good idea, added.